docs/src/proposals/optimistic-confirmation-and-slashing.md

```diff
@@ -0,0 +1,89 @@
+# Optimistic Confirmation and Slashing
+
+Progress on optimistic confirmation can be tracked here
+
+https://github.com/solana-labs/solana/projects/52
+
+At the end of May, the mainnet-beta is moving to 1.1, and testnet is moving
+to 1.2. With 1.2, testnet will behave as if we have 1-block conf as
+long as no more than 4.66% of the validators are acting maliciously.
+Applications can assume that 2/3+ votes observed in gossip confirm a
+block or that at least 4.66% of the network is violating the protocol.
+
+## How does it work?
+
+The general idea is that validators have to continue voting, following
+their last fork, unless they can construct a proof that their fork
+may not reach finality. The way validators construct this proof is by
+collecting votes for all the other forks, excluding their own. If
+the set of valid votes represents over 1/3+X of the epoch stake weight,
+there is no way for the validators current fork to reach 2/3+ finality.
+The validator hashes the proof (creates a witness) and submits it
+with their vote for the alternative fork.  If 2/3+ votes for the
+same block, it is impossible for any of the nodes to construct this
+proof, and therefore no node is able to switch forks and this block
+will be eventually finalized.
+
+
+## Tradeoffs
+
+The safety margin is 1/3+X, where X represents the minimum amount
+of stake that will be slashed in case the protocol is violated. The
+tradeoff is that liveness is now reduced by 2X in the worst case.
+If more than 1/3 - 2X of the network is unavailable, the network may
+stall and will resume finalizing blocks after the network recovers.
+So far, we haven’t observed a large unavailability hit on our
+mainnet, cosmos, or tezos. For our network, which is primarily
+composed of high availability systems, this seems unlikely. Currently,
+we have set the threshold percentage to 4.66%, which means that
+23.68% of the nodes have to be available for the network to continue
```

Comment:
33.333 - 4.6666 * 2 = 24%

Also I think this should read `at most 24% of nodes can be malicious`,  or `at least 76%` of nodes have to be correct.

`24%` is the maximum failure number, not the availability number